### PR TITLE
docs: add just eat kongverge integration to kong hub

### PR DESCRIPTION
**Pull request by Gadam8**
### Summary

A PR to get the Just Eat Kongverge integration into Kong Hub. This includes the addition of a Just Eat publisher directory and a Kongverge extension in the _app/_hub directory. A logo was also added. 

### Full changelog

* added index.md file for just eat kongverge integration in docs
* added logo for just eat kongverge integration in docs

### Checklist:

- [x] [Commit message & atomicity](https://github.com/Kong/docs.konghq.com/blob/master/CONTRIBUTING.md#commit-atomicity) checked
- [ ] Documentation 
- [x] Spellchecked my updates
- [x] Ready to be merged

### CODE REVIEW

File: app/_hub/justeat/kongverge/index.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+#        - 0.8.x
+#        - 0.7.x
+#        - 0.6.x
+#        - 0.5.x
+#        - 0.4.x
+#        - 0.3.x
+#        - 0.2.x
+    enterprise_edition:
+      compatible:
+        - 0.33-x
```
Comment (coopr) [inline app/_hub/justeat/kongverge/index.md]: I just want to check on this statement of compatibility with Kong Enterprise 0.33 - that version is based on Kong 0.13.1 per https://docs.konghq.com/enterprise/changelog/#033---20180724 - and I see above that you don't declare compatibility with Kong 0.13.x.

Can you clarify with what versions of Kong and Kong Enterprise you've successfully used Kongverge? Thanks!

File: app/_hub/justeat/kongverge/index.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+#        - 0.30-x
+#        - 0.29-x
+
+###############################################################################
+# END YAML DATA
+# Beneath the next --- use Markdown (redcarpet flavor) and HTML formatting only.
+#
+# The remainder of this file is for free-form description, instruction, and
+# reference matter.
+# Your headers must be Level 3 or 4 (parsing to h3 or h4 tags in HTML).
```
Comment (coopr) [inline app/_hub/justeat/kongverge/index.md]: These instructions in the template are wrong-ish - headers must start at Level 2, and must be properly nested - sorry for not updating that before you created this PR!

File: app/_hub/justeat/kongverge/index.md [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+# This is represented by ### or #### notation preceding the header text.
+###############################################################################
+# BEGIN MARKDOWN CONTENT
+---
+
+### Documentation
```
Comment (coopr) [inline app/_hub/justeat/kongverge/index.md]: Please update to heading level 2.

File: app/_hub/justeat/kongverge/index.md (hunk 1 above)
Comment (Gadam8) [reply app/_hub/justeat/kongverge/index.md]: Thanks for your reply! 

Kongverge is compatible with CE 0.14.0 onwards, we have tested with 0.13.0 & 1 and some of our unit tests fail. I have checked the change log documentation for Kong Enterprise versions and the latest version (0.33-2) inherits from CE 0.13.0 & 1, does this render Kongverge incompatible with any Kong EE versions?

Hope this makes sense!

File: app/_hub/justeat/kongverge/index.md (hunk 2 above)
Comment (Gadam8) [reply app/_hub/justeat/kongverge/index.md]: No worries, I'll update accordingly.

File: app/_hub/justeat/kongverge/index.md (hunk 3 above)
Comment (Gadam8) [reply app/_hub/justeat/kongverge/index.md]: I will update accordingly.

File: app/_hub/justeat/kongverge/index.md (hunk 1 above)
Comment (coopr) [reply app/_hub/justeat/kongverge/index.md]: Yes, I believe that Kongverge would currently be incompatible with all released versions of Kong Enterprise. Future Kong Enterprise versions will, eventually, be based on versions of Kong with which Kongverge is compatible.

So, I'd suggest you mark all versions of Kong prior to 0.14, and all versions of Kong Enterprise, as incompatible - thanks.

Comment (Gadam8) [reply app/_hub/justeat/kongverge/index.md]: Okay, thank you for your help. I have updated and committed the changes.

File: app/_hub/justeat/kongverge/index.md [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+
+license_type: Apache-2.0
+
+license_url: https://github.com/justeat/kongverge/blob/master/LICENSE
+
+kong_version_compatibility:
+    community_edition:
+      compatible:
+        - 0.14.x
+#      incompatible:
```
Comment (coopr) [inline app/_hub/justeat/kongverge/index.md]: This compatibility information won't appear if it is commented out - I'll adjust that now...

**Comment by coopr** [comment]
Thanks for much for contributing to the Kong Hub - well done @Gadam8 !!

**Comment by coopr** [comment]
Well shoot, now that I merged this and tried running or deploying the docs site, I'm getting an error:

`Error: could not read file /Users/cooper/Documents/GitHub/docs.konghq.com/app/_hub/justeat/kongverge/index.md: (<unknown>): did not find expected key while parsing a block mapping at line 2 column 1`

And I'm having trouble fixing this - any ideas @Gadam8 ? I can't see anything wrong in your file that would cause this.

**Comment by Gadam8** [comment]
Thanks for the merge. Line 2 column 1 is where the name is in the file, I can't see any issues there. The file seems fine, the only thing I can think of is the compatibility for EE is present but has no values?

Apologies for the errors.

**Comment by coopr** [comment]
@Gadam8 I'm not exactly sure what I did, but thanks for the clue - the issue was indeed somewhere in the compatibility section, despite that unhelpful error message :)

It seems that https://github.com/Kong/docs.konghq.com/pull/1017 fixes it - I'll merge and deploy now, you should see your new listing on https://docs.konghq.com/hub/ shortly!

**Comment by Gadam8** [comment]
I'm glad it's all resolved, thanks for doing that. Apologies for all the back and forth :)

That's brilliant, how exciting! Thank you for your time.

**Pull request merged**